spec/controllers/sign_up/passwords_controller_spec.rb

```diff
@@ -87,7 +87,7 @@
                 count: Devise.password_length.first,
               )],
             password_confirmation:
-              ["is too short (minimum is #{Devise.password_length.first} characters)"],
+              [I18n.t('errors.messages.too_short.other', count: Devise.password_length.first)],
```

Comment:
I think we can leverage Rails' [default pluralization logic](https://guides.rubyonrails.org/i18n.html#pluralization) rather than explicitly referencing the non-`one` key:

```suggestion
              [I18n.t('errors.messages.too_short', count: Devise.password_length.first)],
```